scripts/docker/deps/compile_llvm_r25.sh

```diff
@@ -3,8 +3,8 @@ LLVM_TARGET="AArch64;X86"
 echo "LLVM Android Version: $(git --git-dir=/LLVM/.git describe --dirty)"
 
 cmake -GNinja -S /LLVM/llvm -B /tmp/build_llvm_ndk                                                                   \
-      -DCMAKE_CXX_COMPILER=clang++-11 \
-      -DCMAKE_C_COMPILER=clang-11 \
+      -DCMAKE_CXX_COMPILER=clang++-14 \
+      -DCMAKE_C_COMPILER=clang-14 \
       -DCMAKE_CXX_FLAGS="-stdlib=libc++" \
```

Comment:
I think we need to use the GCC standard library with Clang here, I keep getting linking issues otherwise (`undefined reference to __sF'`). Does that sound good to you, @weliveindetail?